### DISCUSSION TITLE
How to draw Bitmap (screenshot) videos in Android? (black when playing videos)

### DISCUSSION THREAD

**Discussion by lwmirkk** [airsdk/Adobe-Runtime-Support#706] [Q&A]
Hi!

I use AIR 33.1.1.245. (Android)
Tested in many versions of AIR.

I can take screenshot from almost everything, but I can not take screenshot from videos playing in full screen. (the screenshot is fiull with black color).

Has anyone ever experienced this?

Thanks a lot.

**Comment by itlancer** [comment]
There is a bug about that: https://github.com/Gamua/Adobe-Runtime-Support/issues/87

**Reply by lwmirkk** [reply]
@itlancer  I need to talk to you, if you can send me an email or give me your email.

my email:
lwirkk @ hotmail [dot] com

Thanks a lot. =)
